protocol/0019-MCAL-margin_calculator.md

```diff
@@ -436,6 +436,9 @@ Assume a [capped future](./0093-CFUT-product_builtin_capped_future.md) market wi
 - Party A posts an order to sell `20` contracts at a price of `17`. A trade is generated for `10` contracts at a price of `18` with party B. A sell order for `10` contracts at a price of `17` from party A gets added to the book. The maintenance and initial margin levels for party A is now `10 * (100 - 17) = 830`, the position is `0` and the remaining volume on the book from this party is `10` at a price of `18`. Party A lost `120` on its position, hence `830 - (300 - 120) = 410` additional funds get moved from the general account as part of the transaction which submitted the order to sell `20` at `17`. Party B now has a position of `0` and following orders open on the book: sell `5` at `20` and buy `30` at `16`. The maintenance and initial margin levels are `max(5 * (100 - 20), 30 * 16) = 480`. The margin account momentarily becomes `820` (`700` + `120` of gains from the now closed position of `-10`), order margin account balance is `480`, hence `820` gets released back into the general account and margin account becomes `0`. (<a name="0019-MCAL-158" href="#0019-MCAL-158">0019-MCAL-158</a>)
 - `fully-collateralised mode` is only allowed when there is a `max_price` specified (<a name="0019-MCAL-170" href="#0019-MCAL-170">0019-MCAL-170</a>)
 - `binary_settlement` is only allowed when there is a `max_price` specified (<a name="0019-MCAL-171" href="#0019-MCAL-171">0019-MCAL-171</a>)
+- When market gets updated so that `fully-collateralised=true` all parties should have their margin brought to the initial level implied by this mode. Any funds above that value should be released back to the general account. If party was in a cross-margin mode and current margin account balance is now insufficient under the fully-collateralised mode then additional funds should be searched for in the general account. No search should be carried out for parties which were in isolated mode. If parties can't meet the new margin level (irrespective of the mode they were in before) their orders should get cancelled and margin requirements should get recalculated. If the new initial margin level still can't be met then they should get closed-out. (<a name="0019-MCAL-177" href="#0019-MCAL-177">0019-MCAL-177</a>)
+- When market gets updated so that `fully-collateralised=false` then all parties should be in cross-margining mode. All parties should have their margin brought to the initial level implied by this mode. Any funds above that value should be released back to the general account. f parties can't meet the new margin level their orders should get cancelled and margin requirements should get recalculated. If the new initial margin level still can't be met then they should get closed-out. (<a name="0019-MCAL-178" href="#0019-MCAL-178">0019-MCAL-178</a>)
```

Comment:
```suggestion
- When market gets updated so that `fully-collateralised=false` then all parties should be in cross-margining mode. All parties should have their margin brought to the initial level implied by this mode. Any funds above that value should be released back to the general account. If parties can't meet the new margin level their orders should get cancelled and margin requirements should get recalculated. If the new initial margin level still can't be met then they should get closed-out. (<a name="0019-MCAL-178" href="#0019-MCAL-178">0019-MCAL-178</a>)
```

Reply:
AC removed now